src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Circle extends ColorFigure {
+    private double radiusSquare;
+
+    public Circle(String color, double radiusSquare) {
+        super(color);
+        this.radiusSquare = radiusSquare;
+    }
+
+    public double getRadiusSquare() {
+        return radiusSquare;
+    }
+
+    public void setRadiusSquare(double radiusSquare) {
+        this.radiusSquare = radiusSquare;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: , circle:  "
+                + getArea() + " sq. units, " + "radiusSquare: "
+                + radiusSquare + " units, color: " + getColor());
+    }
+
+    public double getArea() {
+        return Math.PI * (radiusSquare * radiusSquare);
```

Comment:
The calculation of the area is incorrect. Since `radiusSquare` is already the square of the radius, the area should be calculated as `Math.PI * radiusSquare`. Adjust this calculation to reflect the correct formula for the area of a circle.